docs/types.rst

```diff
@@ -26,4 +26,6 @@ tuple with a second ``bool`` value denoting success.
 
 .. include:: types/operators.rst
 
+.. include:: types/literal-suffixes.rst
```

Comment:
Not sure what's the best place for it. I shoved it somewhere between "operators", and "literal conversions" because it's related, but this whole page seems overloaded with stuff that's somewhat related to types but could also be put into a category of its own.

Reply:
I think it's fine for the time being; we can 'refactor' the page later.